readme/BREAKING_CHANGES.markdown

```diff
@@ -480,4 +480,26 @@ The `frontend-taglib-clay` module is now using components from
 [`Clay v3`](https://github.com/liferay/clay), which doesn't support the
 previous attributes.
 
+---------------------------------------
+### Remove CSS Compatibility Layer
+- **Date:** January 2021
```

Comment:
I think Brian is going to ask for a `YYYY-Jan-DD` format here.

Reply:
done!